From e8a7bae0a750bcb5d8aadc45629d2c9adaf2106c Mon Sep 17 00:00:00 2001
From: Vadim Zeitlin <vadim@wxwidgets.org>
Date: Sun, 21 Jul 2019 21:22:55 +0200
Subject: [PATCH] Alternative clang on Windows link fix

Remove WXDLLIMPEXP_BASE from wxObjectEventFunctor declaration: this
class only has inline methods and so doesn't define any functions that
would need to be exported from the DLL.

---
 include/wx/event.h   | 2 +-
 1 files changed, 1 insertion(+), 1 deletions(-)

diff --git a/include/wx/event.h b/include/wx/event.h
index c196b72b2754..c13e7336b159 100644
--- a/include/wx/event.h
+++ b/include/wx/event.h
@@ -210,7 +210,7 @@ class WXDLLIMPEXP_BASE wxEventFunctor
 };
 
 // A plain method functor for the untyped legacy event types:
-class WXDLLIMPEXP_BASE wxObjectEventFunctor : public wxEventFunctor
+class wxObjectEventFunctor : public wxEventFunctor
 {
 public:
     wxObjectEventFunctor(wxObjectEventFunction method, wxEvtHandler *handler)
